src/cpp/session/modules/NotebookHtmlWidgets.R

```diff
@@ -133,6 +133,11 @@
    # force a responsive viewer sizing policy
    x$sizingPolicy$viewer.padding <- 0
    x$sizingPolicy$viewer.fill <- TRUE
+
+   # make width dinamic to size correctly non-figured
+   if (!is.null(x$sizingPolicy$knitr) && identical(x$sizingPolicy$knitr$figure, FALSE)) {
```

Comment:
Just FYI (no need to change this), recently learned from @kevinushey that this can be rewritten simply as `identical(x$sizingPolicy$knitr$figure, FALSE)` since in R `NULL$foo` is a legal expression that evaluates to `NULL` without error, so there's no need for the tower of `NULL` checks as is necessary in most languages when following an object chain.